force-app/main/default/objects/npe01__OppPayment__c/fields/RefundType__c.field-meta.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>RefundType__c</fullName>
+    <description>Determine if the record is fully or partially refunded</description>
+    <externalId>false</externalId>
+    <inlineHelpText>Determine if the record is fully or partially refunded</inlineHelpText>
+    <label>Refund Type</label>
+    <required>false</required>
+    <trackFeedHistory>false</trackFeedHistory>
+    <trackTrending>false</trackTrending>
+    <type>Picklist</type>
+    <valueSet>
+        <valueSetDefinition>
+            <sorted>false</sorted>
+            <value>
+                <fullName>Refund</fullName>
+                <default>false</default>
+                <label>Refund</label>
+            </value>
+            <value>
+                <fullName>Partial Refund</fullName>
+                <default>false</default>
+                <label>Partial Refund</label>
```

Comment:
@andrewyu-salesforce please also add one more value to this picklist: `Chargeback`